mos_tests/environment/os_actions.py

```diff
@@ -83,6 +89,11 @@ def __init__(self, controller_ip, user='admin', password='admin',
                                                  endpoint_type='publicURL')
         token = self.session.get_token()
         self.heat = HeatClient(endpoint=endpoint_url, token=token)
+
+        murano_endpoint = self.session.get_endpoint(
+            service_type='application_catalog', endpoint_type='publicURL')
+        self.murano = MuranoClient(endpoint=murano_endpoint, token=token,
```

Comment:
If MuranoClient support initialization with session, let's use it (like GlanceClient, for example)